### PR TITLE
fix: after bulk generate, retain tests

### PR DESCRIPTION
**Pull request by saravmajestic**
### Problem
After bulk generating documentation, if a model has tests for columns, tests were deleted

### Solution
While saving the documentation after bulk generate, send tests data from webview to retain the tests

### Screenshot/Demo

NA

### How to test
- Open a model with tests in columns
- Bulk generate documentation
- save 
- after reloading vscode, tests should be retained

## Checklist

- [x] I have run this code and it appears to resolve the stated issue
- [ ] `README.md` updated and added information about my change

### CODE REVIEW

File: webview_panels/src/modules/documentationEditor/components/docGenerator/GenerateButton.tsx [hunk 1]
```diff
@@ -37,7 +39,11 @@ const GenerateButton = ({
     }
   };
   return (
-    <Button color="primary" onClick={onSubmit} title="Generate description">
+    <Button
```
Comment (saravmajestic) [inline webview_panels/src/modules/documentationEditor/components/docGenerator/GenerateButton.tsx:42]: added for e2e tests

**Review by AdiGajbhiye — APPROVED** [review]
lgtm

File: webview_panels/src/modules/documentationEditor/components/saveDocumentation/SaveDocumentation.tsx [hunk 2]
```diff
@@ -26,7 +26,7 @@ const SaveDocumentation = (): JSX.Element | null => {
   const saveDocumentation = async () => {
     const result = (await executeRequestInSync("saveDocumentation", {
       ...currentDocsData,
-      updatedTests: isTestUpdatedForAnyColumn ? currentDocsTests : undefined,
+      updatedTests: currentDocsTests,
```
Comment (AdiGajbhiye) [inline webview_panels/src/modules/documentationEditor/components/saveDocumentation/SaveDocumentation.tsx:29]: i am assuming this is the only change which retains test by passing the current tests. rest all changes are for tests.

**Review by mdesmet — APPROVED** [review]

File: webview_panels/src/modules/documentationEditor/components/saveDocumentation/SaveDocumentation.tsx (hunk 2 above)
Comment (saravmajestic) [reply webview_panels/src/modules/documentationEditor/components/saveDocumentation/SaveDocumentation.tsx:29]: yes, right @AdiGajbhiye

**Pull request merged**